das/local_disk_das.go

```diff
@@ -8,34 +8,43 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/offchainlabs/nitro/arbutil"
-	"github.com/offchainlabs/nitro/solgen/go/bridgegen"
 	"os"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+
 	"github.com/offchainlabs/nitro/arbstate"
+	"github.com/offchainlabs/nitro/arbutil"
 	"github.com/offchainlabs/nitro/blsSignatures"
+	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"github.com/offchainlabs/nitro/solgen/go/bridgegen"
+
 	flag "github.com/spf13/pflag"
 )
 
 var ErrDasKeysetNotFound = errors.New("no such keyset")
 
 type LocalDiskDASConfig struct {
-	KeyDir                string `koanf:"key-dir"`
-	PrivKey               string `koanf:"priv-key"`
-	DataDir               string `koanf:"data-dir"`
-	AllowGenerateKeys     bool   `koanf:"allow-generate-keys"`
-	L1NodeURL             string `koanf:"l1-node-url"`
-	SequencerInboxAddress string `koanf:"sequencer-inbox-address"`
-	StorageType           string `koanf:"storage-type"`
+	KeyDir                string               `koanf:"key-dir"`
+	PrivKey               string               `koanf:"priv-key"`
+	DataDir               string               `koanf:"data-dir"`
+	S3Config              genericconf.S3Config `koanf:"s3"`
+	RedisConfig           RedisConfig          `koanf:"redis"`
+	BigCacheConfig        BigCacheConfig       `koanf:"big-cache"`
+	AllowGenerateKeys     bool                 `koanf:"allow-generate-keys"`
+	L1NodeURL             string               `koanf:"l1-node-url"`
```

Comment:
`l1-node-url` and `sequencer-inbox-address` from this object and `AggregatorConfig` should be moved up to `DataAvailabilityConfig` so there is no duplication of parameters between storage and aggregator parameters.